eduNEXT/xblock-mindmap
# fix: add instructions for learners

### Description
This PR adds instructions for students and instructors to use the mind map component.

### How To Test
1. Install this branch in your environment.
2. Access a unit in LMS/Studio.
3. You should see the instructions to use the mind map component as a drop-down menu.
### Screenshots

1. LMS
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/10f2342f-7080-4d78-943e-2fcc26359d08)
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/90482f0f-30c0-48dc-9f01-30397c3180ff)
4. Studio
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/a9c8dbb7-aa79-4695-bcd8-238731500c1a)
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/5ff359c4-e961-47d6-ba06-bdb079854457)
